src/index.ts

```diff
@@ -398,7 +398,7 @@ export const plugin: PluginFunction<TypescriptMocksPluginConfig> = (schema, docu
                         transformUnderscore,
                     });
 
-                    return `        ${fieldName}: overrides && overrides.hasOwnProperty('${fieldName}') ? overrides.${fieldName}! : ${value},`;
+                    return `        ${fieldName}: overrides?.${fieldName} || ${value},`;
```

Comment:
I wonder if this can break integrations on apps that doesn't support optional chaining?
Maybe we need to define a peerDependency with minimal typescript version?